vk-mobile-nav/package/_scss/_sub_item_accordion.scss

```
@@ -14,11 +14,24 @@
 	li {
 		position: relative;
 		margin-bottom: 0;
-		&.menu-item-has-children{
-			& > a {
-				padding-right:3em;
-			}
-		}
+	}
+	// 子階層を持つ項目は grid 化し、タイトルと開閉ボタンを横並びにする。
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)